public/manifest.json

```diff
@@ -19,5 +21,11 @@
     "default_popup": "popup.html"
   },
   "manifest_version": 2,
-  "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'"
+  "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'",
+  "browser_specific_settings": {
+    "gecko": {
+        "id": "sendtokodi@firsttris.github.io"
+    }
+  }
```

Comment:
Unfortunately this is required to use the `chrome.storage.sync` API in Firefox. I say "unfortunately " because Chrome doesn't recognize this key and so throws a (non-fatal?) warning.

Reply:
Just took a closer look. With this key, Chrome says there's an error when the extension is loaded but the extension works fine otherwise.

Reply:
Solution: https://github.com/firsttris/chrome.sendtokodi/pull/3/files#diff-52e55511a7f447a6d0c08a6d3f6351cd51034f8d161016c585e3325af4431afeR11-R17

I'm open to other approaches, this is just the first thing that came to mind.